src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/update-alert-notification-channels.mdx

```diff
@@ -25,6 +28,32 @@ When you make updates, you can rename a channel, add policies to an existing cha
 Here's a quick reference for updating channels which also includes links to more detailed information and procedures.
 
 <CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="view-email"
+    title="View email channels"
+  >
+    Alerts automatically includes the [email addresses](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/notification-channels-controlling-where-send-alerts#email) for all users in the selected account as individual notification channels. If the account is a [child account](/docs/accounts/original-accounts-billing/original-users-roles/mastersub-account-structure), the list shows only the users on the child account, not the users in the parent account.
+    
+    To view or search the list of user names and emails: Go to **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")**, in the top nav click **Alerts & AI**, then click **Notification channels**. In the search field, search for 'user'.
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="add-update-email"
+    title="Add or update email channels"
+  >
+    Users can't unsubscribe from email notifications. The account Owner or Admin must remove them from the policy's notification channel.
+
+To add or update account users as notification channels for a policy:
```

Comment:
I love these collapsers! For easier skimming, I wonder if it would help to arrange them in alpha order or by type of function (add, view, change, delete)? Also, I see "Add email channel" as a collapser title, but at first glance I didn't see how to Add any other type of notification channel. So I assumed "Create more channels" is the procedure for Add others. But it's third from bottom so I didn't immediately notice it, and I had to click open that collapser to make sure it had the info I was looking for. But I may be too picky with this wording question. Take it or leave it! :-)

Reply:
Good call!